common/djangoapps/config_models/models.py

```diff
@@ -136,9 +145,9 @@ def current(cls, *args):
         return current
 
     @classmethod
-    def is_enabled(cls):
+    def is_enabled(cls, *args):
```

Comment:
Yeah, I think this would have made sense with the last approach given I was piping it through all over the third_party_auth code base. Luckily it only happens in two or three places now including tests. :)